src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -50,12 +54,13 @@ public void end(boolean interrupted) {
   public boolean isFinished() {
     if(m_ShooterSubsystem.wheelReady()){
       SmartDashboard.putBoolean("wheelReady", true);
-      if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
-        m_ShooterSubsystem.runCargo(true);
+      //if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
```

Comment:
Disabled because there is no limelight...Should we consider instead returning 0.0 from calculatePID?

Reply:
It would be cleaner, but a harder change to remember, I would have no idea what was going wrong.